e2e-tests/portal/cypress/integration/journeys/maker/dashboard/gef-deals/dashboard-table-data.spec.js

```diff
@@ -71,3 +74,102 @@ context('View a deal', () => {
     });
   });
 });
+
+context('Sort exporters alphabetically', () => {
+  let deals;
+  // eslint-disable-next-line no-unused-vars
+  let exporters;
```

Comment:
do we need this variable here?

Reply:
if I remove it, it complains that it is exporter is not defined and linting errors without the disable

Reply:
you can remove it from here. and below you can say
`cy.insertManyGefExporters(deals, mockExporters, MAKER_LOGIN);`

^^^ you don't need the `.then((insertedExporter) => { exporters = insertedExporter; });` because exporters is not being used anywhere else

Reply:
done